dali/ft/filecopy.cpp

```diff
@@ -22,7 +22,9 @@
 #include "jlib.hpp"
 #include "jio.hpp"
 #include <math.h>
-
+#include <azure/core.hpp>
+#include <azure/storage/blobs.hpp>
+#include <azure/storage/files/shares.hpp>
```

Comment:
related to @ghalliday 's question above - should this be on a conditional compile switch (USE_AZURE) ?

Reply:
On an offline discussion, Gavin mentioned that it is not necessary to handle optional components for vcpkg packages as should always be available. @ghalliday @jakesmith